### DISCUSSION TITLE
Adding a "screen reader text" block option via themes

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#181] [Topic Ideas]
Currently, the best way to mark a Heading, Paragraph, or other block to only be shown to screen readers is to add the `.screen-reader-text` class via the **Advanced** tab. 

What if there was a more user-friendly way? Maybe a simple toggle switch in the block settings?

Building off the [Beyond Block Styles](https://developer.wordpress.org/news/tag/beyond-block-styles/) tutorial series, I'd like to explore how we can add something a little simpler that could have a potentially big impact for folks who need the ability to mark text for screen readers only.

**Comment by justintadlock** [comment]
Discussion is moved to the issue after approval. Please continue the conversation there 😃

- https://github.com/WordPress/developer-blog-content/issues/193

**Comment by bph** [comment]
@justintadlock [closed the issue](https://github.com/WordPress/developer-blog-content/issues/193#issuecomment-2271700856) 

I unlocked the discussion if someone would like to explore the topic further.

**Reply by bph** [reply]
Locked it again and closed it, too.
